### PR TITLE
Refactor post_authors indexer to avoid repeated calls to the catalog

### LINKED ISSUE #56: Refactor post_authors indexers

### PR DESCRIPTION
**Pull request by ericof**
Fix #56

### CODE REVIEW

File: src/collective/blog/indexers/post.py [hunk 1]
```diff
@@ -15,12 +15,7 @@ def post_authors(obj: Post):
...
-        searchResult = api.content.find(portal_type="Author", UID=uid)
-        if len(searchResult) > 0:
-            author = searchResult[0].getObject()
-
-            if author is not None:
-                authors.append(author.title)
-
+    brains = api.content.find(portal_type="Author", UID=uids)
+    for brain in brains:
+        authors.append(brain.title)
```
Comment (davisagli) [inline src/collective/blog/indexers/post.py:20]: The catalog metadata has Title capitalized:

```suggestion
        authors.append(brain.Title)
```

**Review by davisagli — CHANGES_REQUESTED** [review]

**Comment by ericof** [comment]
Good call, I will update the PR

**Comment by jonaspiterek** [comment]
I tested it locally and the upgrade step was broken, pushed a fix.

**Review by jonaspiterek — COMMENTED** [review]
LGTM

**Comment by davisagli** [comment]
@jonaspiterek thanks, good catch

**Pull request merged**